packages/glib/meson.build.patch

```diff
@@ -8,7 +8,15 @@
      '-Werror=implicit-function-declaration',
      '-Werror=init-self',
      '-Werror=missing-include-dirs',
-@@ -2047,9 +2046,8 @@
+@@ -511,7 +510,6 @@
+   'getresuid',
+   'getvfsstat',
+   'gmtime_r',
+-  'hasmntopt',
```

Comment:
It seems like there is no way to tell meson that this function is not available, incase someone knows a cleaner approach, feel free to suggest